apps/new-dealer/models/submission.js

```diff
@@ -2,10 +2,27 @@
 /* eslint complexity: 0 max-statements: 0 */
 const contains = (arr, val) => arr.indexOf(val) > -1 ? 'Yes' : 'No';
 
+const authorityType = usage => {
+  if (usage.indexOf('arm-guards') > -1) {
+    return 'Maritime Guards';
+  }
+
+  if (usage.indexOf('transport') > -1 || usage.indexOf('transfer') > -1) {
```

Comment:
I don't think I agree, abstracting this kind of logic into a "helper" function just means you end up having to go and look somewhere else to work out what the actual logic is, and if anything makes the code harder to follow.